.gitattributes

```diff
@@ -1,5 +1,8 @@
 * text=auto eol=lf
 
+/docs/sites.yml merge=union
```

Comment:
Would we ever have yaml files we wouldn't want this behavior to apply to?

Reply:
IMO sidebar links should cause merge conflicts that need to be manually resolved, so 👍 for selective merge strategy just for showcase entries

Reply:
Yeah, I was thinking more explicit is better until we play with things a bit. And yeah, good point on sidebar links merge conflicts potentially being problematic.